CONTIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to the ObjectScript extension for Visual Studio Code
+
+## Contributing a pull request
+
+### Prerequisites
+
+1. [Node.js](https://nodejs.org/) 12.x
+1. Windows, macOS, or Linux
+1. [Visual Studio Code](https://code.visualstudio.com/)
+1. The following VS Code extensions:
+    - [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+    - [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+    - [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)
+
+### Setup
+
+```shell
+git clone https://github.com/intersystems-community/vscode-objectscript
+cd vscode-objectscript
+npm install
+```
+
+### Errors and Warnings
+
+TypeScript errors and warnings will be displayed in the `Problems` window of Visual Studio Code.
+
+### Run dev build and validate your changes
+
+To test changes, open the `vscode-objectscript` folder in VSCode.
+Then, open the debug panel by clicking the `Run and Debug` icon on the sidebar, select the `Launch Extension` or `Launch Extension Alone`
```

Comment:
Suggest adding an explanation of the differences between these two launch configurations.

Reply:
I've done this in https://github.com/intersystems-community/vscode-objectscript/pull/183/commits/a9f0d146dde1125fd9af6387fea6bfdd08460008